ICAFIX/hcp_fix_multi_run

```diff
@@ -182,6 +189,7 @@ fi
 debug_disable_trap
 set +u
 source "${FSL_FIXDIR}/settings.sh"
+# source "${HCPPIPEDIR}/ICAFIX/settings.sh.WUSTL_CHPC2"
```

Comment:
I'm not sure we want this in here even as a comment.  I'm also not sure how relevant that CHPC2 config file is anymore.